index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Abel mobile setup </title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      <div class="home-page">
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#">Abel</a> 
```

Comment:
- [ ] Please stick to the Figma design but your logo text is not the same as the Figma design your `font size` and `font weight` is not matching with Figma design:+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

| Your Design | Figma Design|
|--------|--------|
| ![Screenshot from 2023-06-09 12-55-44](https://github.com/abel12-pro/Portfolio_mobileFirst/assets/45226848/24ce06a8-f39d-48ba-a301-d01bff9a4b2f) | ![Screenshot from 2023-06-09 12-55-37](https://github.com/abel12-pro/Portfolio_mobileFirst/assets/45226848/54593572-2426-42ba-8a6c-23cb96deff5c) |